vehicle/OVMS.V3/components/ovms_webserver/src/web_cfg.cpp

```diff
@@ -951,8 +951,8 @@ void OvmsWebServer::HandleCfgModem(PageEntry_t& p, PageContext_t& c)
   enable_sms = MyConfig.GetParamValueBool("modem", "enable.sms", true);
   enable_gps = MyConfig.GetParamValueBool("modem", "enable.gps", false);
   enable_gpstime = MyConfig.GetParamValueBool("modem", "enable.gpstime", false);
-  cfg_sq_good = MyConfig.GetParamValueFloat("network", "modem.sq.good", -95);
-  cfg_sq_bad = MyConfig.GetParamValueFloat("network", "modem.sq.bad", -93);
+  cfg_sq_good = MyConfig.GetParamValueFloat("network", "modem.sq.good", -93);
```

Comment:
Though this is specific to the web server, so shouldn't affect us. 
Still, leaving this comment here to flag it for future just in case. 
I don't understand why they've swapped these two values.